scripts/collect-assets.js

```diff
@@ -22,11 +25,14 @@ const RELEASE_TAG = "v0.0.1";
 const DOWNLOAD_DIR = _resolve(__dirname, "../downloaded_assets");
 const META_FILE = _resolve(DOWNLOAD_DIR, "collect.json");
 
+// Release asset filenames and nodejs equivalent platform/arch
+//  platform: https://nodejs.org/docs/latest-v22.x/api/process.html#processplatform
+//  arch: https://nodejs.org/docs/latest-v22.x/api/process.html#processarch
 const ASSETS_TO_DOWNLOAD = [
-  { name: "kai-rpc-server.linux-x86_64.zip", folder: "linux", platform: "linux" },
-  { name: "kai-rpc-server.macos-arm64.zip", folder: "macos-arm", platform: "macos-arm" },
-  { name: "kai-rpc-server.macos-x86_64.zip", folder: "macos-x86", platform: "macos-x86" },
-  { name: "kai-rpc-server.windows-x86_64.zip", folder: "windows", platform: "windows" },
+  { name: "kai-rpc-server.linux-x86_64.zip", platform: "linux", arch: "x64", chmod: true },
+  { name: "kai-rpc-server.macos-arm64.zip", platform: "darwin", arch: "arm64" },
+  { name: "kai-rpc-server.macos-x86_64.zip", platform: "darwin", arch: "x64" },
```

Comment:
I think we need to add +x to the binaries for mac as well unless I am missing something.